open-machine-learning-jupyter-book/ml-fundamentals/regression/loss-function.ipynb

```diff
@@ -57,7 +57,7 @@
    "id": "f5ee754a-e5d9-4532-86d5-1433722be122",
```

Comment:
<p>There should be a 'your turn' and 'acknowledge' section after that.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/590/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='74'/>